pkg/gen/input/workflows/internal/file/create_release.go

```diff
@@ -28,6 +28,7 @@ on:
   push:
     branches:
       - 'legacy'
+	  - 'main'
```

Comment:
This just made all workflows fail. Please no tabs no more :heart:

Reply:
:facepalm: On it